lib/utils/markdown.js

```diff
@@ -13,6 +13,12 @@ export const markdownToHtml = (data) => {
     match.replace(/<\/?p>/g, "")
   );
 
+  // Convert ![alt](src) to <img src="src" alt="alt">
+  parsedHTML = parsedHTML.replace(
```

Comment:
@AbhayVAshokan _a A doubt here. Isn't there a possibility of the input having multiple occurrence of this kind of image? In that case, shouldn't we use `replaceAll` instead of replace?